apps/system/js/utility_tray.js

```diff
@@ -250,22 +248,12 @@ window.UtilityTray = {
           return;
         }
 
-        // Prevent swipe down gesture when already opened/opening.
-        if (target !== this.grippy && this.showing) {
-          return;
-        }
-
-        if (target !== this.overlay && target !== this.grippy &&
-            evt.currentTarget !== this.statusbarIcons &&
-            evt.currentTarget !== this.topPanel) {
-          return;
+        if (this._isDraggable(target)) {
+          this.onTouchStart(evt.touches[0]);
+        } else {
+          evt.stopPropagation();
         }
 
-        if (this._shouldPrevent(target)) {
```

Comment:
It's likely to stop bad behaviour with the events falling through to the app underneath, or some other such thing (Etienne might know better iirc) - Either way, I wouldn't remove it if it doesn't have any effect on what you're trying to do without first knowing why it's there.